src/components/Users/UsersContainer.js

```diff
@@ -1,38 +1,88 @@
+import React from 'react';
 import { connect } from 'react-redux';
 
+import Preloader from '../../utils/preloader';
+
+import axios from 'axios';
+
 import {
-  followActionCreator,
-  unfollowActionCreator,
-  setUsersActionCreator,
-  setCurrentPageActionCreator,
-  setTotalUsersCountActionCreator,
+    followActionCreator,
+    unfollowActionCreator,
+    setUsersActionCreator,
+    setCurrentPageActionCreator,
+    setTotalUsersCountActionCreator,
+    toggleIsFetchingActionCreator
 } from '../../redux/actions/actionCreators';
 
-import UsersAPIComponent from './UsersAPIComponent';
+import Users from './Users';
+
+class UsersContainer extends React.Component {
+
+    componentDidMount() {
+        this.props.toggleIsFetching(true);
+        axios.get(`https://social-network.samuraijs.com/api/1.0/users?page=${this.props.currentPage}&count=${this.props.pageSize}`)
+            .then(response => {
+                this.props.toggleIsFetching(false);
+                this.props.setUsers(response.data.items);
+                this.props.setTotalUsersCount(response.data.totalCount);
+            });
+    };
+
+    onPageChanged = (pageNumber) => {
+        this.props.toggleIsFetching(true);
+        this.props.setCurrentPage(pageNumber);
+        axios.get(`https://social-network.samuraijs.com/api/1.0/users?page=${pageNumber}&count=${this.props.pageSize}`)
+            .then(response => {
+                this.props.toggleIsFetching(false);
+                this.props.setUsers(response.data.items);
+            });
+    };
+
+    render() {
+        return (
+            <>
+                {this.props.isFetching ? <Preloader /> : null}
+                < Users totalUsersCount={this.props.totalUsersCount}
```

Comment:
The indentation here seems off.
Don't forget to run "npm run lint"

Reply:
/fxd